vignettes/excel-editor.Rmd

```diff
@@ -0,0 +1,97 @@
+---
+title: "Editing a Dataset from the Excel Editor"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Editing a Dataset from the Excel Editor}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(crplyr)
+```
+
+# What is it?
+- Elevator pitch: (DP tool, allows preparing a dataset's structure in a familiar way: MS Excel)
+- (what is crunch automation?)
+- Brief note on terminology: Mostl uses Crunch Automation terminology, which conflicts with rcrunch (title, label)
+  (I should probably change Alias to Name, shouldn't I?)
```

Comment:
No we're keeping Title and Alias. We could touch up the rcrunch “variable by any other name” bit to refer to Title but we decided it would be too disruptive to try to pretend we got that right lo forever ago.

Reply:
Gotcha, mostly asked because having `RENAME` mean change alias is a little confusing.